zhaquirks/xiaomi/aqara/sensor_switch_aq3.py

```diff
@@ -56,6 +59,17 @@
 }
 
 
+class OppleCluster(XiaomiAqaraE1Cluster):
+    """Opple cluster."""
+
+    attributes = {
+        0x0009: ("mode", types.uint8_t, True),
+    }
+    attr_config = {0x0009: 0x01}
+
+    async def bind(self):
+        """Bind cluster."""
```

Comment:
Since `SKIP_CONFIGURATION` is `True`, this won't be called anyway.
So, you can remove it completely.

Reply:
it is removed